openapi-module/README.md

```diff
@@ -0,0 +1,97 @@
+
+# OpenAPI Module for Flexbench
+
+This module provides functionalities to parse OpenAPI documents and generate test scenarios for Flexbench.
+
+## Prerequisites
+
+- To use this module, you need to install Node.js. It is recommended to use the latest LTS version available.
+  Please install Node.js using pre-built installers for your platform to avoid incompatibility issues with different development tools.
+
+### To check that Node.js was installed correctly, type the following commands in your terminal client:
+
+```sh
+node -v
+npm -v
+```
+
+## Project Structure
+
+```
+openapi-module/
+├── config/
+├── sample/
+│   ├── mock-server.js
+│   └── sample-openapi.yaml
+├── scripts/
+│   ├── generate-curl.js
+│   └── generate-flex.js
+├── src/
+│   ├── generators/
+│   │   ├── curl-generator.js
+│   │   ├── fake-data.js
+│   │   └── flex-generator.js
+│   └── parsers/
+│       └── openapi-parser.js
+ temp/
+│   ├── curl-commands.sh
+│   └── flex-scenarios.flex
+└── test/
+    └── parser.test.js
+.gitignore
+main.js
+package-lock.json
+package.json
+README.md
+```
+
+## Installation
+
+### To install dependencies, run the following command:
```

Comment:
@AJun01 I am following the instructions in my local env and I get this error:

```
iskitsas001@GR_D36WYRX1MV openapi-module % npm run generate-curl --openApiFilePath=sample/sample-openapi.yaml --outputFilePath=temp/curl-commands.sh

> flexbench-openapi-module@1.0.0 generate-curl
> node scripts/generate-curl.js

/Users/iskitsas001/gitRepos/flexivian/flexbench/openapi-module/scripts/generate-curl.js:4
const openApiFilePath = args.find(arg => arg.startsWith('--openApiFilePath=')).split('=')[1];
                                                                              ^

TypeError: Cannot read properties of undefined (reading 'split')
    at Object.<anonymous> (/Users/iskitsas001/gitRepos/flexivian/flexbench/openapi-module/scripts/generate-curl.js:4:79)
    at Module._compile (node:internal/modules/cjs/loader:1255:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1309:10)
    at Module.load (node:internal/modules/cjs/loader:1113:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12)
    at node:internal/main/run_main_module:23:47

Node.js v20.2.0
iskitsas001@GR_D36WYRX1MV openapi-module %
```

Commands I ran through terminal:

```
cd openapi-module 
npm install
npm run generate-curl --openApiFilePath=sample/sample-openapi.yaml --outputFilePath=temp/curl-commands.sh
```

Reply:
fixing it now

Reply:
Fixed: OPENAI_API_KEY=****** npm run generate-flex --  --openApiFilePath=sample/sample-openapi.yaml --outputFilePath=temp/flex-scenarios.json